plotly/tests/test_core/test_graph_objs/test_annotations.py

```diff
@@ -25,33 +24,28 @@ def test_trivial():
     assert Annotations() == list()
 
 
-@raises(PlotlyError)
 def test_weird_instantiation():  # Python allows this, but nonsensical for us.
-    print(Annotations({}))
+    assert Annotations({}) == list()
 
 
 def test_dict_instantiation():
     Annotations([{'text': 'annotation text'}])
 
 
-@raises(PlotlyDictKeyError)
 def test_dict_instantiation_key_error():
-    print(Annotations([{'not-a-key': 'anything'}]))
+    assert Annotations([{'not-a-key': 'anything'}]) == [{'not-a-key': 'anything'}]
```

Comment:
OK I see, the new way to do this would be to just use a regular list of `go.layout.Annotation`:
```
[go.layout.Annotation(**{'not-a-key': 'anything'})]
```

Reply:
So, we should probably update this test name as well (we can do that in the end, once we've gone through the rest of the tests)